tests/e2e/contact-summary.js

```diff
@@ -224,4 +296,31 @@ describe('Contact summary info', () => {
       )
     ).toBe('1');
   });
+
+  it('should display UHC Stats in contact summary, if contact counts visits and user has permission', async () => {
+    await utils.createUsers([ USER ]);
```

Comment:
I had to add this login because it turned out the logged in user was admin. What do you think about this e2e?